include/ofxImageEffect.h

```diff
@@ -993,17 +994,20 @@ See the documentation on clip preferences for more details on how this is used w
 #define kOfxImageUnPreMultiplied "OfxImageAlphaUnPremultiplied"
 
 /** @brief Indicates whether an effect should premultiply or not on output.
-This provides clarification for generators or effect where input is kOfxImageOpaque.
+Added in version 1.5: This provides clarification for generators or effect where input is kOfxImageOpaque.
 
 - Type - string X 1
-- Property Set - a plugin  instance (read only)
-- Default - 0
+- Property Set - a plugin  instance (read only). Ideally this information should already be available at parameter creation stage.
 Valid Values - This must be one of
 
 - kOfxImagePreMultiplied   - the image is premultiplied by its alpha
 - kOfxImageUnPreMultiplied - the image is unpremultiplied
 
-This Property defines an host level policy.
+This Property defines an host level policy. It is for practical purposes considered global in the scope of a Project.
+
+Recommended Interpretation: If an host adds support for OfxImageEffectHostDesiredPremult, the host should set on the output the same value on output clip kOfxImageEffectPropPreMultiplication preference 
```

Comment:
This needs some grammar work.

Reply:
:) - upping the standard now

Also should not say Recommended but Expected here I guess